.github/workflows/master.yml

```diff
@@ -247,7 +247,7 @@ jobs:
       - uses: ./.github/actions/pipdeptree
       - name: Run tests
         run: |
-          pytest --splits=${{ matrix.splits }} --group=${{ matrix.group }} tests/models
+          pytest --splits=${{ matrix.splits }} --group=${{ matrix.group }} --serve-wheel tests/models
```

Comment:
Could you add this to pyfunc and evaluate tests? They have also failed (due to same root cause)
https://github.com/mlflow/mlflow/actions/runs/6677805050/job/18148176342?pr=10207

Reply:
Absolutely!

Reply:
added